SourceCode/Power/ftt_p_main.py

```diff
@@ -810,6 +808,9 @@ def solve(data, time_lag, iter_lag, titles, histend, year, domain):
             data['MWIY'][:, :, 0] = (data_dt['MWIY'][:, :, 0]
                                      + (data['MEWI'][:, :, 0] * dt * data['BCET'][:, :, c2ti['3 Investment ($/kW)']] / 1.33))
             
+            sector_coupling_assumps = get_sector_coupling_dict(data, titles)
+            
+            cumulative_capacity_power = data["MSSC"] * sector_coupling_assumps["GW to GWh"]
```

Comment:
This is a good start, but this computes the current capacity. What we need to compute is the cumulative capacity, by having computing the yearly additions:
* Compute the difference between the capacity of this year and the previous year (data["MSSC"]) ... - time_lag["MSSC"] ..., 
* Add the yearly replacements of existing capacity (when they reach their end of life). That is: add data["MSSC"]/life_time_battery.

Reply:
Got it! Thanks.